colab/nessie-iceberg-demo-nba.ipynb

```diff
@@ -9,7 +9,12 @@
     "This demo showcases how to use Nessie Python API along with Spark3 from Iceberg\n",
     "\n",
     "Initialize Pyspark + Nessie environment\n",
-    "----------------------------------------------"
+    "----------------------------------------------\n",
+    "In order to get started, we will first have to do a few setup steps that give us everything we need\n",
+    "to get started with Nessie. The `nessiedemo` lib configures all required dependencies, downloads and\n",
+    "configures Spark...\n",
+    "In case you're interested in the detailed setup steps that Spark requires, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
```

Comment:
```suggestion
    "In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
```